Gems/Camera/Code/Source/ViewportCameraSelectorWindow.cpp

```diff
@@ -161,8 +173,13 @@ namespace Camera
             return index(row, 0);
         }
 
+        void CameraListModel::Reset()
+        {
+            m_lastActiveCamera.SetInvalid();
+            m_cameraItems = AZStd::vector<CameraListItem>(1, CameraListItem(AZ::EntityId()));
```

Comment:
Suggestion: might be nice to add a comment explaining why m_cameraItems defaults to a list of 1 item with an invalid EntityId instead of an empty list.

Reply:
This predates me and is to handle the normal 'Editor Camera' (not tied to an Entity/Component) in the level - I'll add a comment to explain 👍